DSharpPlus/Entities/DiscordGuild.cs

```diff
@@ -370,7 +376,7 @@ public DiscordRole EveryoneRole
         /// </summary>
         [JsonProperty("is_owner", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
I believe this is obsoleted with `owner`, so this JsonProperty attribute should be removed.

Reply:
Actually my mistake, there is a better solution for this. All you need to do is change `is_owner` to `owner` in this JsonProperty.

Reply:
Why is it even there? `owner_id` isn't anywhere in the docs (its get only, so it wouldn't be used in deserialization anyway)

Reply:
And no, it's get only right now, so I'd have to add backing field aswell. (right?)

Reply:
> Why is it even there? owner_id isn't anywhere in the docs (its get only, so it wouldn't be used in deserialization anyway)

The `owner` property isn't present when [getting a guild through rest](https://discord.com/developers/docs/resources/guild#get-guild), which is why both are used.

Reply:
I know that, and also I know that owner id isn't present when getting guild by https://discord.com/developers/docs/resources/user#get-current-user-guilds, which is why this pr was made (duh).
I'm talking about original question of this change request - JsonProperty attribute

Reply:
Oh I see what you mean now. Yeah I'm really sorry for the confusion then lol, ignore my newest requests and go with what I originally requested.

> I'm talking about original question of this change request - JsonProperty attribute

It isn't needed to serialize anymore because it was obsolete anyway with the change to `owner` and because it is now serialized in your internal property.

Reply:
And you should probably replace it with a `JsonIgnore` attribute as well.